app/models/observation.rb

```diff
@@ -258,9 +258,14 @@ class Observation < AbstractModel
         -> { where.not(name: Name.unknown) }
   scope :without_name,
         -> { where(name: Name.unknown) }
-  scope :without_confident_name, lambda {
-    without_name.or(where(vote_cache: ..0))
+  scope :with_name_above_genus,
+        -> { where(name_id: Name.with_rank_above_genus.map(&:id)) }
```

Comment:
```suggestion
        -> { where(name_id: Name.with_rank_above_genus.ids) }
```
a wee bit shorter

Reply:
Thanks @JoeCohen, i was completely unaware of that method.  I'm having trouble finding available documentation of it, and it's hard to google. 

APIDock implies it's an alias for `pluck`, in which case I don't think we would want to use it. In my long research detour on this subject a few months ago, I found that `pluck` was definitively slower than `map`. (`select` was way slower too.)

Please let me know if you're able to find out what it does.